cluster/ubuntu/util.sh

```diff
@@ -359,6 +360,21 @@ function kube-up() {
   create-kubeconfig
 }
 
+function create-systemd-service() {
```

Comment:
I just following the pattern that other services is using:
put the service file into /lib/systemd/system, and make a soft link to /etc/systemd/system

And copy these service files directly into "/etc/systemd/system" also work.